core/src/main/java/nu/yona/server/Translator.java

```diff
@@ -46,4 +44,24 @@ public String getLocalizedMessage(String messageId, Object[] parameters, Locale
 
 		return msgSource.getMessage(messageId, parameters, locale);
 	}
+
+	public static String getStandardLocaleString(Locale locale)
+	{
+		return locale.toString().replace('_', '-');
+	}
+
+	public static String buildLocaleSpecificResourcePath(String format)
+	{
+		return MessageFormat.format(format, determineLocaleInfix());
+	}
+
+	private static Object determineLocaleInfix()
+	{
+		Locale locale = LocaleContextHolder.getLocale();
+		if (locale.equals(Locale.forLanguageTag("en-US")))
+		{
+			return "";
```

Comment:
It's not the subfolder but the extension of the file name (before the dot). The default files contain US English and don't have such a "suffix". E.g. ``message.properties``, while the Dutch version is ``message_nl.properties``.